obspy/clients/eida/fetch.py

```diff
@@ -0,0 +1,735 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+################################################################################
+# (C) 2014-2016 Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ #
+#                                                                              #
+# License: LGPLv3 (https://www.gnu.org/copyleft/lesser.html)                   #
+################################################################################
+
+"""
+A command-line FDSN Web Service client with EIDA routing and authentication support.
+"""
+import sys
+import time
+import datetime
+import optparse
+import threading
+import socket
+import csv
+import re
+
+try:
+    # Python 3.2 and earlier
+    from xml.etree import cElementTree as ET
+
+except ImportError:
+    from xml.etree import ElementTree as ET
+
+try:
+    # Python 2.x
+    import Queue
+    import urllib2
+    import urlparse
+    import urllib
+
+except ImportError:
+    # Python 3.x
+    import queue as Queue
+    import urllib.request as urllib2
+    import urllib.parse as urlparse
+    import urllib.parse as urllib
```

Comment:
In principle we use `python-future` for most of these, although we're not really happy with that approach either. So I guess it can stay like this..

In any case, using `from future.utils import PY2; if PY2: ... else: ...) would be cleaner than the `try/except` for now..